dist/stat_data.py

```diff
@@ -303,6 +303,9 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_hs_key_truncate_mix_ts_restart', 'history store key truncation calls that returned restart'),
     CacheStat('cache_hs_ondisk', 'history store table on-disk size', 'no_clear,no_scale,size'),
     CacheStat('cache_hs_ondisk_max', 'history store table max on-disk size', 'no_clear,no_scale,size'),
+    CacheStat('cache_hs_order_fixup_insert', 'history store table out-of-order updates that were fixed up during insertion'),
+    CacheStat('cache_hs_order_fixup_move', 'history store table out-of-order updates that were fixed up by moving existing records'),
+    CacheStat('cache_hs_order_resolved', 'history store table out-of-order resolved updates that lose their durable timestamp'),
```

Comment:
Is it possible to add durable timestamp lost in the stat name? How about `cache_hs_order_lose_durable_timestamp`.

Reply:
ea0294d

Thanks. That is a better name.